ui/src/app/cohort-search/search-group-list/search-group-list.component.html

```diff
@@ -1,22 +1,14 @@
 <div class="search-groups-header">
   <h3>{{ title | titlecase }}</h3>
-  <div class="add-group">
-    <button
-      type="button"
-      class="btn btn-success-outline"
-      [style.margin-right]="'0'"
-      (click)="initGroup()">
-      <clr-icon shape="plus"></clr-icon> 
-      Add Group
-    </button>
-  </div>
 </div>
 
 <!-- .size, not .length => this is an immutable.js list -->
-<div *ngIf="(groups$ | async)?.size > 0" class="search-groups">
+<div class="search-groups">
   <div *ngFor="let group of (groups$ | async); let i = index">
-    <div *ngIf="i > 0" class="text-center font-weight-bold">AND</div>
+    <!--<div *ngIf="i > 0" class="text-center font-weight-bold">AND</div>-->
```

Comment:
If this is dead code let's remove it.  Also it looks like we're no longer using the `i` from `i = index`, we can remove that part as well.

Reply:
Done.